### PR TITLE
[17.0] [MIG] asterisk_click2dial: Migration to 17.0

### PR DESCRIPTION
**Pull request by luisDIXMIT**
removed js after considering it unnecessary

@etobella @alexis-via

### CODE REVIEW

**Review by etobella — APPROVED** [review]

**Comment by cvinh** [comment]
@luisDIXMIT can you apply changes of v16 please 
https://github.com/OCA/connector-telephony/pull/319
And maybe rebase from v16... it would be easier to review

**Comment by luisDIXMIT** [comment]
@cvinh comments looked after. Migration done from v16

File: asterisk_click2dial/models/asterisk_server.py [hunk 1]
```diff
@@ -169,10 +169,7 @@ def test_ari_connection(self):
             "tag": "display_notification",
             "params": {
                 "type": "success",
-                "message": _(
-                    "Connection Test Successfull! Odoo can successfully login to"
-                    "Asterisk."
-                ),
+                "message": _("Your request has been successfully sent"),
```
Comment (cvinh) [inline asterisk_click2dial/models/asterisk_server.py]: Same as for v16, please keep the original message

Comment (luisDIXMIT) [reply asterisk_click2dial/models/asterisk_server.py]: You're right. I have updated it.

**Review by cvinh — APPROVED** [review]

**Pull request merged**